packages/flutter_markdown/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.6.11
+
+* Fix shrinkWrap not taken into account with single child #105299
```

Comment:
I'm guessing the `#105299` at end of line is referring to the bug fixed by this change. I'd personally prefer this to be `flutter/flutter#105299` or even `flutter/flutter Issue #105299` as that is not ambiguous.

Reply:
I've added a link and prefixed it with: [https://github.com/flutter/flutter/issues/105299](https://github.com/flutter/flutter/issues/105299) (same syntax GitHub uses).

I've also rewritten the message to use a "present tense indicative for verbs" as indicated in the [changelog style guidelines](https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style).